aggregate/webtools-site/pom.xml

```diff
@@ -47,28 +47,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <!-- Workaround for publish script not able to deal with ${artifactId}-${version}.zip -->
```

Comment:
I see this was removed in multiple places.

Looks great not having to have this kind of thing, but just so I grok it right - this has nothing to do with sha1/md5 sum generation changes, correct ?